Casks/m/mpluginmanager.rb

```diff
@@ -0,0 +1,31 @@
+cask "mpluginmanager" do
+  version "02.15"
+  sha256 "541d06685c530279c2ee71051c3b04c4141a47acac46504e461ab2c404ed57c5"
+
+  url "https://www.meldaproduction.com/downloads/down?name=MPluginManager_#{version.dots_to_underscores}_setupmac.zip&platform=mac&version=#{version}&mirror=bunnycdn&url=https%3A%2F%2Fmeldaproduction.b-cdn.net%2Fdownload%2Fmpluginmanager%2FMPluginManager_#{version.dots_to_underscores}_setupmac.zip&checksum=8a265945c2b263f916e622791cce101372fee1fd"
+  name "MPluginManager"
+  desc "Installer for MeldaProduction audio plugins"
+  homepage "https://www.meldaproduction.com/downloads"
+
+  livecheck do
+    url :homepage
+    regex(/href=.*?platform=mac.*?version=v?(\d+(?:[._-]\d+)*)/i)
```

Comment:
```suggestion
    regex(/href=.*?MPluginManager[._-]v?(\d+(?:[._]\d+)+)[._-]setupmac\.zip/i)
    strategy :page_match do |page, regex|
      page.scan(regex).map { |match| match[0].tr("_", ".") }
    end
```

The `_setupmac` suffix in the zip file name should be enough to make sure that we're matching the Mac version, so we can use a more standard regex pattern. for matching the version from the file name. We also need to use a `strategy` block to replace `_` in the version string with `.`, to match the cask `version` format.